src/routes/index.js

```diff
@@ -28,7 +28,26 @@ router.put('/users/:id/firstname', catchErrors(userController.setUserFirstname))
 router.get('/users/:id/lastname', catchErrors(userController.getUserLastName));
 //Set Last Name
 router.put('/users/:id/lastname', catchErrors(userController.setUserLastname));
+<<<<<<< Updated upstream
 // GET Specific User Emai;
 router.get('/user/:id/email', catchErrors(userController.getUserEmail));
+=======
+
+//Get Phone number
+router.get('/users/:id/phoneNumber', catchErrors(userController.getphoneNumber));
```

Comment:
`phone` is okay, no need for `phoneNumber`

Reply:
I used it because the user property was "phoneNumber".

I'll change it.